styles.css

```diff
@@ -14,4 +14,23 @@ body {
 .container {
     width: 700px;
     margin: 0 auto;
+}
+
+main {
+    width: 100%;
+    padding: 10px;
+    overflow: auto;
+    background-image: linear-gradient(#288FB0, #295074);
+    border-radius: 10px;
+    margin: 10px 0 10px 0;
+}
+
+.card {
+    width: 150px;
```

Comment:
this isn't doing anything either

Reply:
I think it's worth defining the box sizes here - they won't be set anywhere else.